### ISSUE TITLE
Using Zesty version 0.2.23 on El Capitan causes parts of the Finder-bar to become opaque

### ISSUE THREAD

**Issue by Skyyblaze** [w0lfschild/zestyWin#2]
Hello,

I'm not sure if Zesty is still in development and I know you said there are bugs so it's no big deal but I thought I would let you know that Zesty 0.2.23 (mySIMBL identifies it as 0.2.22) causes the entire right part of the finder bar to become opaque as can be seen in my screenshot. I'm running El Capitan 10.11.6 (15G1217) with mySIMBL version 0.5.1 (28). If there's anything I can do to help troubleshooting and fixing that please let me know.

![bar](https://cloud.githubusercontent.com/assets/22203339/22353009/bdc13d6c-e41e-11e6-935e-845a144ed327.png)

EDIT: It turns out that I was running 0.2.22 even though it was labeled as 0.2.23 on GitHub but even compiling 0.2.23 with Xcode which then also shows up correctly as 0.2.23 in mySIMBL produces the same issue.

**Comment by w0lfschild** [comment]
Yeah it's still in development in the sense that I will fix bugs that get reported and are simple to fix.

I'll fix this at some point for sure.

**Comment by Skyyblaze** [comment]
Okay alright thanks. I really like the look Zesty gives OSX as a whole. I also noticed a opaque box around my user-avatar in the login-screen by now but I suppose that's easily fixed by blacklisting the loginwindow process, I'll try that later.

**Comment by PatrikTheDev** [comment]
I have the same problem on high Sierra, what should I do? I really like this plugin but it is broken
